source/Octopus.Client/OctopusAsyncClient.cs

```diff
@@ -135,126 +140,37 @@ public static async Task<IOctopusAsyncClient> Create(OctopusServerEndpoint serve
         private static async Task<IOctopusAsyncClient> Create(OctopusServerEndpoint serverEndpoint, OctopusClientOptions options, bool addHandler)
         {
             var client = new OctopusAsyncClient(serverEndpoint, options ?? new OctopusClientOptions(), addHandler);
-            try
-            {
-                var rootResource = await client.EstablishSession().ConfigureAwait(false);
-                var spaceRootResource = await LoadSpaceRootResource(client, rootResource);
-                client.RootDocuments = new RootResources(rootResource, spaceRootResource);
-                client.Repository = new OctopusAsyncRepository(client);
-                return client;
-            }
-            catch
-            {
-                client.Dispose();
-                throw;
-            }
-        }
-
-        private static async Task<SpaceRootResource> LoadSpaceRootResource(OctopusAsyncClient client, RootResource rootResource)
-        {
-            if (client.clientOptions.SpaceContext == null)
-            {
-                var defaultSpace = await GetDefaultSpace(client, rootResource);
-                client.clientOptions.SpaceContext = defaultSpace != null ? SpaceContext.SpecificSpaceAndSystem(defaultSpace.Id) : SpaceContext.SystemOnly();
-            }
-
-            var spaceId = GetSpaceId(client.SpaceContext);
-            return !string.IsNullOrEmpty(spaceId) ?
-                await client.Get<SpaceRootResource>(rootResource.Link("SpaceHome"), new { spaceId }).ConfigureAwait(false)
-                : null;
-        }
-
-
-        /// <summary>
-        /// Gets a document that identifies the Octopus server (from /api) and provides links to the resources available on the
-        /// server. Instead of hardcoding paths,
-        /// clients should use these link properties to traverse the resources on the server. This document is lazily loaded so
-        /// that it is only requested once for
-        /// the current <see cref="IOctopusAsyncClient" />.
-        /// </summary>
-        public RootResource RootDocument => RootDocuments.RootResource;
-
-        public SpaceRootResource SpaceRootDocument => RootDocuments.SpaceRootResource;
-
-        RootResources RootDocuments { get; set; }
-
-        class RootResources
-        {
-            public RootResources(RootResource rootResource, SpaceRootResource spaceRootResource)
-            {
-                RootResource = rootResource;
-                SpaceRootResource = spaceRootResource;
-            }
-
-            public RootResource RootResource { get; }
-            public SpaceRootResource SpaceRootResource { get; }
+            // User used to see this exception 
+            // System.PlatformNotSupportedException: The handler does not support custom handling of certificates with this combination of libcurl (7.29.0) and its SSL backend
+            await client.Repository.LoadRootDocument();
```

Comment:
Ah this is annoying. Is there a better way to do this? Or can we remove this switch altogether now?